docs/sources/plugins/_index.md

```diff
@@ -4,5 +4,48 @@ type = "docs"
 [menu.docs]
 name = "Plugins"
 identifier = "plugins"
-weight = 8
+weight = 1
 +++
+
+# Plugins
+
+In addition to the wide range of visualizations and data sources that are available immediately after you install Grafana, you can extend your Grafana experience through _plugins_.
+
+You can [install]({{< relref "./installation.md" >}}) one of the plugins built by the Grafana community, or [build one yourself]({{< relref "./developing/development.md" >}}).
+
+Grafana supports three types of plugins: _panels_, _data sources_, and _apps_.
+
+## Panel plugins
+
+Add new visualizations to your dashboard through panel plugins, such as [world maps](https://grafana.com/grafana/plugins/grafana-worldmap-panel), [clocks](https://grafana.com/grafana/plugins/grafana-clock-panel), and [pie charts](https://grafana.com/grafana/plugins/grafana-piechart-panel).
```

Comment:
```suggestion
Add new visualizations to your dashboard with panel plugins, such as [world maps](https://grafana.com/grafana/plugins/grafana-worldmap-panel), [clocks](https://grafana.com/grafana/plugins/grafana-clock-panel), and [pie charts](https://grafana.com/grafana/plugins/grafana-piechart-panel).
```

Reply:
I generally recommend that we use the exact name of the item linked as the link text.